### ISSUE TITLE
Validate actual HTML sent by server, not the parse tree

### ISSUE THREAD

**Issue by lkmorlan** [mgueury/html_validator#55]
If I attempt to validate a page containing an un-escaped angle bracket, it will say it is OK. I presume this is because it validates the parse tree with the angle bracket already escaped as an entity. The actual online validator raises an error.

```
<!DOCTYPE html>
<html><head>
<meta charset="utf-8">
<title>Title</title>
</head><body>
<div>></div>
</body></html>
```

**Comment by mgueury** [comment]
To do: log a bug in Tidy

**Comment by mgueury** [comment]
Thanks for reporting the issue. I tried to run the HTML file via the validator.v3.com 
but it returns no error ?
Was there a issue when you copy/pasted the HTML into the editor above ?

If yes, maybe send me a mail with the file to mgueury@skynet. be

Thanks

Ps: result for the validator.w3.org site:

Document checking completed. No errors or warnings to show.
Source

    <!DOCTYPE html>↩
    <html><head>↩
    <meta charset="utf-8">↩
    <title>Title</title>↩
    </head><body>↩
    <div>></div>↩
    </body></html>

**Comment by mgueury** [comment]
Note, when redoing the test. Be sure to say if you use or Firefox or Chrome when doing the testing.
The result could be different. There is more chance that it works better in Chrome due to issue #11.

**Comment by lkmorlan** [comment]
That is strange. I'm sure I got an error when I tried it before posting this issue. Still, in the HTML Validator view of the source, it it shows up as `&gt;` even though the source actually contains `>`.

I get the error if I use a less-than. The following today produces an error on `validator.w3.org/nu` but not in HTML Validator where it shows it as `&lt;`.

```
<!DOCTYPE html>
<html><head>
<meta charset="utf-8">
<title>Title</title>
</head><body>
<div><</div>
</body></html>
```
